civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -706,8 +706,8 @@ function civihr_employee_portal_permission() {
       'title' => t('Manage HR Reports configuration'),
       'description' => t('Availability to manage HR Reports configuration'),
     ],
-    'civihr - access root menu items and configurations' => array(
-      'title' => t('CiviHR - Access root menu items and configurations'),
+    'access root menu items and configurations' => array(
+      'title' => t('CiviHR: Access root menu items and configurations'),
       'description' => t('Access advanced configuration options that could potentially impair site functionality')
```

Comment:
since you're adding the permission in the civicrm extension, you don't need to add it here as well